gdcdictionary/schemas/_definitions.yaml

```diff
@@ -158,15 +158,57 @@ project_id:
     term:
         $ref: "_terms.yaml#/project_id"
 
-data_file_properties:
+ubiquitous_properties: #All entities except for program and project
   id:
     $ref: "#/UUID"
     systemAlias: node_id
   submitter_id:
     type:
       - string
       - "null"
-    description: "The file ID assigned by the submitter." # TOREVIEW
+    description: "The identifier assigned by the submitter."
+  state:
+    $ref: "#/state"
+  project_id:
+    $ref: "#/project_id"
+  created_datetime:
+    $ref: "#/datetime"
+  updated_datetime:
+    $ref: "#/datetime"
+  type:
```

Comment:
Is it worth having a hand written list of properties that must be updated every time you add a node? I feel like `type: string` works better considering the way validation works. If you enter the type name wrong it will fail because it doesn't know what schema you're asking to validate against.

Reply:
Ok, wasn't sure because saw a mix of approaches, so went with what seemed safer on the surface.

Reply:
Yeah its a bit of a hodgepodge with the GDC. I used it a bit on the nodes themselves because it was pretty easy to self-maintaining (build a node and fill out the type), but adding a validation done in a place no one will think to look seems like 🙅‍♂️

Reply:
Ok fixed